package.json

```diff
@@ -26,7 +26,8 @@
                 ],
                 "filenamePatterns": [
                     "suite*.rc",
-                    "suite*.rc.*"
+                    "suite*.rc.*",
+                    "*.cylc"
```

Comment:
Is this okay? Or do we not want the same grammar for `global.cylc` as for `flow.cylc`, in which case it should be changed to `flow*.cylc`

Reply:
Fine.